src/probnum/filtsmooth/statespace/sde.py

```diff
@@ -301,7 +322,9 @@ def _increment_fun(time, mean, cov, driftfun, jacobfun, dispmatfun):
     jacobian = jacobfun(time)
     mean_increment = driftfun(time, mean)
     cov_increment = (
-        cov @ jacobian.T + jacobian @ cov.T + dispersion_matrix @ dispersion_matrix.T
+        cov @ jacobian.T
+        + jacobian @ cov.T
+        + diffusion * dispersion_matrix @ dispersion_matrix.T
```

Comment:
The last "leaf". It should be correct this way since diffusion = sigma^2, but please double-check.

Reply:
I think this is correct